### PR TITLE
add some info and propose some dates

### PR DESCRIPTION
**Pull request by dmbch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,26 @@
-# proposals
\ No newline at end of file
+# HH.js Talk Proposals
+
+If you would like to give a talk at [HH.js](https://www.meetup.com/hamburg-js/), thank you very much! We really appreciate your willingness to put your time and effort into something that will make our meetup more interesting and inspiring.
+
+## Proposals
+
+Please [open an issue](https://github.com/hamburg-js/proposals/issues/new) in this repository. In that issue, we ask you to outline the topic of your talk and to pick a couple of possible dates. Oh, and please let us know if you have already given it before.
```
Comment (herschel666) [inline README.md]: It'd would imho make sense to add `?template=talk-proposal.md` to the "New issue"-URL and to merge PR #1 first then.

File: README.md [hunk 2]
```diff
@@ -1 +1,26 @@
...
+- Monday, September 23rd, location and talks needed!
+- Wednesday, November 20th, location and talks needed!
+
+## Technical Requirements
+
+We generally expect our hosts to provide Wifi, a video projector with HDMI/MiniDP connectors and a speaker microphone. If you have additional requirements (e.g. computer audio), please mention these in your proposal.
+
+## Contact
+
+If you would like to get in touch before actually submitting a talk, you are also welcome, of course. Please use our [Meetup page](https://www.meetup.com/hamburg-js/), join our #hhjs group in the open [WebHH Slack](https://web-hh-slackin.herokuapp.com/) or ping us on [Twitter](http://twitter.com/hhjs).
```
Comment (herschel666) [inline README.md]: We could link `#hhjs group` to `https://web-hh.slack.com/messages/C0KESM96C`, so ppl, that already have an account there, can directly click it.

**Review by herschel666 — APPROVED** [review]
Looks fine content-wise. Just two remarks, which don't block merging in any way.

**Review by ZauberNerd — APPROVED** [review]
Looks good to me!

**Pull request merged**